docs/micrometer.md

```diff
@@ -48,10 +48,10 @@ for {
 // res0: Resource[Task, Server[Task]] = Bind(
 //   Bind(
 //     Bind(
-//       Suspend(zio.ZIO$FlatMap@34ede267),
-//       com.avast.sst.jvm.execution.ExecutorModule$$$Lambda$1652/815927219@6b92a0d1
+//       Suspend(zio.ZIO$FlatMap@56554365),
+//       com.avast.sst.jvm.execution.ExecutorModule$$$Lambda$1652/1769513770@2f2dc407
 //     ),
-//     cats.effect.Resource$$Lambda$1653/888293905@51e0f2eb
+//     cats.effect.Resource$$Lambda$1653/1432768646@320770d7
```

Comment:
is there a way to prevent these useless changes in the future?

Reply:
https://scalameta.org/mdoc/docs/modifiers.html#silent ?

Reply:
pls use it - this is horrible